db/migrate/20180217030453_create_poems.rb

```diff
@@ -1,8 +1,8 @@
 class CreatePoems < ActiveRecord::Migration[5.1]
   def change
     create_table :poems do |t|
-      t.string :greek_title, default: 'Untitled', null: false
-      t.string :english_title, default: 'Χωρίς τίτλο', null: false
+      t.string :greek_title, default: 'Χωρίς τίτλο', null: false
+      t.string :english_title, default: 'Untitled', null: false
```

Comment:
Remember that migrations are strictly additive. After a migration has been merged to master, we never want to change it, because anyone who has already applied the original migration won't get any of the changes when they run `bin/rails db:migrate`. Instead, create a new migration and use the [`change_column_default`](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_default) method in it.